tests/sentry/api/endpoints/test_user_authenticator_details.py

```diff
@@ -203,7 +214,10 @@ def test_recovery_codes_regenerate(self):
         resp = self.client.get(url)
         assert old_codes != resp.data['codes']
 
-    def test_delete(self):
+        assert email_log.info.call_count == 0
```

Comment:
Wouldn't we want this action to send an email?

Reply:
Ya, I'll add it in a follow-up pr ^